rpc/core/tx.go

```diff
@@ -99,6 +101,22 @@ func TxSearch(ctx *rpctypes.Context, query string, prove bool, page, perPage int
 			Tx:       r.Tx,
 			Proof:    proof,
 		}
+
+		if sortOrder == "asc" {
```

Comment:
https://github.com/tendermint/tendermint/blob/9b5cbb55417dd5f3ecc2d7e1d75c3d774ee61c39/state/txindex/kv/kv.go#L250

should we remove ^ then? to avoid extra sorting

Reply:
@melekes But what if some want in descending order?

Reply:
What do you mean @melekes?

Reply:
Yes sure @melekes